common/setups/rasr/config/am_config.py

```diff
@@ -0,0 +1,77 @@
+__all__ = ["TransitionTdp", "SilenceTdp", "AmRasrConfig"]
+
+from dataclasses import dataclass
+from typing import Optional, Tuple, Union
+
+from sisyphus import tk
+
+import i6_core.am as am
+import i6_core.rasr as rasr
+
+
+@dataclass()
+class _Tdp:
+    loop: Union[float, str]
+    forward: Union[float, str]
+    skip: Union[float, str]
+    exit: Union[float, str]
+
+    def get_tuple(
+        self,
+    ) -> Tuple[
+        Union[float, str], Union[float, str], Union[float, str], Union[float, str]
+    ]:
+        return self.loop, self.forward, self.skip, self.exit
+
+
+@dataclass()
+class TransitionTdp(_Tdp):
+    loop: Union[float, str] = 3.0
+    forward: Union[float, str] = 0.0
+    skip: Union[float, str] = "infinity"
+    exit: Union[float, str] = 0.0
+
+
+@dataclass()
+class SilenceTdp(_Tdp):
+    loop: Union[float, str] = 0.0
+    forward: Union[float, str] = 3.0
+    skip: Union[float, str] = "infinity"
+    exit: Union[float, str] = 20.0
+
+
+@dataclass()
+class AmRasrConfig:
+    state_tying: str = "monophone"
```

Comment:
mh.. If you really want to be fancy, could this be an instance of an `enum` object that holds all available state tying methods as `str`?